fixation_tolerance: 0.1
median_window: 3
number_of_frames_before_destruction: 60
number_of_frames_before_destruction_locked: 60

camera: head_camera_rgb_optical_frame
maximum_person_height: 2.1
minimum_person_height: 0.8
filtering_frame_id: base_link
fixed_frame_id: base_link
odom_frame_id: odom

creation_threshold: 0.5
validation_gate: 1000000
metric_weight: 0.8
recognition_threshold: 0.6
c_learning_rate: 0.8
const_pos_var: 0.5
const_vel_var: 0.5
const_accel_var: 0.5

#odom parameters
alpha_1: 0.05
alpha_2: 0.001
alpha_3: 5.0
alpha_4: 0.05

update_min_d: 0.01
update_min_a: 0.01

